libicyque.c

```diff
@@ -1077,7 +1122,6 @@ icq_unread_message_load_cb(IcyQueAccount *ia, JsonObject *data, gpointer user_da
 		gint i, len = json_array_get_length(persons);
 		if(len == 0) return;
 		if(len > 1) {
-			purple_connection_error(ia->pc, PURPLE_CONNECTION_ERROR_OTHER_ERROR, "Group-Conversations not yet supported. Please post the icyque input of the debug window in a github issue.");

```

Comment:
This caused pidgin to disable my account all the time. I am unsure of whether or not its okay to remove this line.

I didn't notice it before because I was using another purple client.

Reply:
Oh, whoops, I mistook this for a debug error not connection error.  Probably would have done something about it sooner for #29 if I had known :(